vectordatasource/transform.py

```diff
@@ -9321,7 +9321,9 @@ def unpack_viewpoint_claims(shape, props, fid, zoom):
     admin_level = str(props.get('tz_admin_level', ''))
 
     if admin_level:
-        base_kind = _ADMIN_LEVEL_TO_KIND.get(admin_level)
+        base_kind = _ADMIN_LEVEL_TO_KIND.get(admin_level, 'region')
+        if not base_kind:
+            base_kind = 'blah'
```

Comment:
we are going to remove this once we figure out the issue here

Reply:
Tell me more about the issue?

Reply:
I believe I fixed the issue here: https://github.com/tilezen/vector-datasource/pull/2054/commits/5afa684b54de99fde44d4c489140458091173b7e

We're using tz_admin_level to determine admin_level, but for any claims that didn't have a claimed_by or disputed_by, we weren't exporting it.